tests/cmdline/commands/test_code.py

```diff
@@ -273,6 +273,27 @@ def test_code_export(run_cli_command, aiida_code_installed, tmp_path, file_regre
     assert isinstance(new_code, InstalledCode)
 
 
+@pytest.mark.usefixtures('aiida_profile_clean')
```

Comment:
I think you can use parametrization to include the sort/not-sort variations in the existing `test_code_export` test. Something like

```python
@pytest.mark.usefixtures('aiida_profile_clean')
@pytest.mark.parametrize('sort', (True, False))
def test_code_export(run_cli_command, aiida_code_installed, tmp_path, file_regression, sort):
"""Test export the code setup to str."""
if sort:
    options = ['--sort']
else:
    options = ['--no-sort']
```
I think the `file_regression` should be clever enough to create separate reference files for each parametrized test, but that I am not a 100% sure off. Pretty sure there should be a way though.

Reply:
Alright, tried this, and it works, `file_regression` is indeed smart enough. Based on my other comment, I'd vouch for the `unsorted` option, though.